purchases_ui_flutter/ios/Classes/PurchasesUiPaywallView.swift

```diff
@@ -0,0 +1,62 @@
+import Flutter
+import UIKit
+import PurchasesHybridCommonUI
+import RevenueCatUI
+
+class PurchasesUiPaywallViewFactory: NSObject, FlutterPlatformViewFactory {
+    private var messenger: FlutterBinaryMessenger
+
+    init(messenger: FlutterBinaryMessenger) {
+        self.messenger = messenger
+        super.init()
+    }
+
+    func create(
+        withFrame frame: CGRect,
+        viewIdentifier viewId: Int64,
+        arguments args: Any?
+    ) -> FlutterPlatformView {
+        return PurchasesUiPaywallView(
+            frame: frame,
+            viewIdentifier: viewId,
+            arguments: args,
+            binaryMessenger: messenger)
+    }
+
+    /// Implementing this method is only necessary when the `arguments` in `createWithFrame` is not `nil`.
+    public func createArgsCodec() -> FlutterMessageCodec & NSObjectProtocol {
+          return FlutterStandardMessageCodec.sharedInstance()
+    }
+}
+
+class PurchasesUiPaywallView: NSObject, FlutterPlatformView {
+    private var _view: UIView
+
+    init(
+        frame: CGRect,
+        viewIdentifier viewId: Int64,
+        arguments args: Any?,
+        binaryMessenger messenger: FlutterBinaryMessenger?
+    ) {
+        if #available(iOS 15.0, *) {
+            let paywallProxy = PaywallProxy()
+            guard let paywallView = paywallProxy.createPaywallView().view else {
+                print("Error: error getting PaywallView.")
+                _view = UIView()
+                super.init()
+                return
+            }
+            paywallView.clipsToBounds = true
```

Comment:
Had to add this otherwise the image in the header of the template would render beyond the screen and would be seen when returning to the previous screen

Reply:
We shouldn't do this. Do you have a screenshot of what it looks like without it?